UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/GasMix.cs

```diff
@@ -211,5 +222,12 @@ private void Recalculate()
 		{
 			Pressure = AtmosUtils.CalcPressure(Volume, Moles, Temperature);
 		}
+
+
+		//Used to change the pressure instead of temperature when removing/Adding gas
+		private void RecalculateTemperatureCache(float _TemperatureCash )
```

Comment:
`float _TemperatureCash` argument is not needed as you always pass TemperatureCache field anyway